src/niscope/metadata/attributes_addon.py

```diff
@@ -12,6 +12,11 @@
     1150128: { 'enum': None, 'python_type': 'bool', }, # ENABLE_TIME_INTERLEAVED_SAMPLING, Don't use the enum because a bool will do
 }
 
+attributes_override_values = {
+    1150085: { 'python_name': 'ADJUST_PRETRIGGER_SAMPLES_5102'.lower(), },
```

Comment:
Seems strange to call lower() rather than to spell out the final python_name. What was the reasoning for this?

Reply:
Less error prone. Copy/paste, then call lower.

Changed.